ProcessList.h

```diff
@@ -48,6 +48,12 @@ typedef struct ProcessList_ {
    Hashtable* displayTreeSet;
    Hashtable* draftingTreeSet;
 
+   struct timeval timestamp;  /* time of the current sample */
+   struct timeval previous;   /* time of the previous sample */
+   uint64_t timestampMs;   /* current time in milliseconds */
+   uint64_t previousMs;		/* previous time in milliseconds */
+   uint64_t intervalMs;		/* milliseconds between 2 samples */
```

Comment:
The fields `previous`, `previousMs` and `intervalMs` are currently only assigned, but never used.
I suppose they are getting used in future additions?

Also, to please my eyes, please align the comments.

Reply:
> The fields previous, previousMs and intervalMs are currently only assigned, but never used.
I suppose they are getting used in future additions?

Yes, should I leave them as-is? or remove them for now?

Reply:
> That's already wrong with the current code flow, as the process update updates some fields for the task counts displayed in the header …

I've opened https://github.com/htop-dev/htop/pull/587 to ensure this bug fix goes in as a standalone fix, as discussed on IRC.